sequential_map_matcher:
 origin_map: '/ABot1/map'
 target_map: '/ABot2/map'
 n_particles: 2000
 n_observation: 1000
 n_history: 4
 n_theta: 50
 n_x: 20
 n_y: 20
 R_var: 0.1
 Q_xy: 0.02
 Q_theta: 0.02
 R_xy: 0.1
 R_theta: 0.1
 P_theta: [0.6, 0.1, 0.1, 0.2]
 P_xy: [0.7, 0.05, 0.05, 0.1, 0.1]
 xy_mul: 2.0
